habitat/tasks/rearrange/actions.py

```diff
@@ -126,6 +129,40 @@ def step(self, vel, should_step=True, *args, **kwargs):
             return None
 
 
+@registry.register_task_action
+class ArmAbsPosAction(SimulatorTaskAction):
+    @property
+    def action_space(self):
+        return spaces.Box(shape=(7,), low=0, high=1, dtype=np.float32)
+
+    def step(self, set_pos, should_step=True, *args, **kwargs):
+        # No clipping because the arm is being set to exactly where it needs to
+        # go.
+        self._sim: RearrangeSim
+        self._sim.robot.arm_motor_pos = set_pos
+        if should_step:
+            return self._sim.step(HabitatSimActions.ARM_ABS_POS)
+        else:
```

Comment:
Else after return is discouraged. Flake8 should complain.

Reply:
Do you mean it should be `return None` without else?